node/src/database/db_migrations.rs

```diff
@@ -373,25 +422,13 @@ mod tests {
                     .map(|str| str.to_string())
                     .collect::<Vec<String>>(),
             );
-            self.execute_upon_transaction_result.borrow_mut().remove(0)
-        }
-
-        fn commit(&mut self) -> Result<(), String> {
-            self.commit_params.lock().unwrap().push(());
-            self.commit_results.borrow_mut().remove(0)
-        }
-
-        fn panic_if_the_version_found_is_too_high(&self, mismatched_schema: usize) {
-            self.panic_if_the_version_found_is_too_high_params
-                .lock()
-                .unwrap()
-                .push(mismatched_schema);
+            self.execute_upon_transaction_results.borrow_mut().remove(0)
         }
     }
 
     lazy_static! {
-        static ref TEST_DIRECTORY_FOR_DB_MIGRATION: PathBuf =
-            PathBuf::from(format!("{}/db_migration", BASE_TEST_DIR));
+        static ref TEST_DIRECTORY_FOR_DB_MIGRATION: String =
```

Comment:
If the forward slash causes problems in Windows, consider modifying this to be the result of some PathBuf arithmetic: that is, start with a PathBuf initialized to BASE_TEST_DIR and .join() db_migration onto it; then PathBuf will take care of deciding between / and \ based on the OS.